src/components/ReplyCard/ReplyCard.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="blip-container reply-card">
+    <div :class="'bubble ' + position">
+      <div class="reply-container">
+        <div class="reply-header">
+          <bds-icon name="undo" theme="outline" aria-label="Reply icon"></bds-icon>
+          <bds-typo tag="p" variant="fs-14" bold="regular" italic="true">{{ translations.reply || 'Resposta' }}</bds-typo>
+        </div>
+        <in-reply-to-base :in-reply-to="inReplyTo" :is-own-message="isOwnMessage" />
+        <replied-base :replied="replied" />
+      </div>
+    </div>
+    <blip-card-date
+      :status="status"
+      :position="position"
+      :date="date"
+      :failed-to-send-msg="failedToSendMsg" />
+  </div>
+</template>
+
+<script>
+  import { default as base } from '../../mixins/baseComponent.js'
+
+  export default {
+    name: 'reply-card',
+    mixins: [base],
+    props: {
```

Comment:
I think `status` and `failedToSendMsg` are missing from the props definition.

Reply:
Fixed